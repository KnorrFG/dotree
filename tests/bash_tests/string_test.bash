$DT -c string_test.dt e
