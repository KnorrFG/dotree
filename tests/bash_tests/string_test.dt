snippet vars = !"
FOO="foo"	
"!

snippet a_fn = " # we want a newline here
append_foo() { 
	echo $1 foo
}
" # and here

menu root {
	e: $vars + "echo foo=$FOO" + $a_fn + "append_foo $FOO"
}
