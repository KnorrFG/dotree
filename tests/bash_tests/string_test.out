[?25l[?25hfoo=foo
foo foo
